c_reference/include/quantized_fastgrnn.h

```diff
@@ -183,7 +187,11 @@ typedef struct Q7xQ15_FastGRNN_Params {
   const Q7_T* mean;
   const Q7_T* stdDev;
   const Q15_T* W;
+  const ITER_T* Wids;
+  const Q15_T* Wvals;
```

Comment:
why are we adding more variables to fastgrnn params struct. would it be better to start another struct for sparse fastgrnn?

Reply:
If we add another `struct` for sparse params, the `fastgrnn()` function will have to be duplicated again, just for changing one line of code:
```
const Q7xQ15_FastGRNN_Params* tparams = (const Q7xQ15_FastGRNN_Params*)params;
```
Hence, I'm avoiding this for now. Folding sparse support into our existing dense framework saves us a lot of duplication.

Reply:
OK, do we want to leave some comments in the struct header file so people using it in dense mode dont worry about these four additional params?